### DISCUSSION TITLE
POST http://netbox.lutech.lab/api/ipam/ip-addresses/ not working

### DISCUSSION THREAD

**Discussion by bbird81** [netbox-community/netbox#8802] [Q&A]
### NetBox version

3.1.7

### Python version

3.8

### Steps to Reproduce

Use the POST method for creating a new ip address

### Expected Behavior

A new ip address should have been created

### Observed Behavior

Netbox always returns "200 OK" with the list of current ip address already present in the IPAM module.

**Comment by bbird81** [comment]
Tried many ways to add an ip address from API:
- using pynetbox nb.ipam.ip_addresses.create method
- using requests to POST on http://netbox/api/ipam/ip-addresses/
- using Insomnia to build the POST

I also tried to copy examples from others (which I assume work for them), but no luck for me...

**Reply by candlerb** [reply]
Can you show the exact command and body, e.g. using curl?  If you're getting back a 200 with a list of IP addresses, it seems Netbox thinks you're doing a GET and not a POST as you expect.

I would suggest you run tcpdump to be 100% sure. On the Netbox server:

```
tcpdump -i eth0 -nn -s0 -A tcp port 8001
```

Replace `eth0` with your interface name (`ip addr list` to find out).  This should show the plaintext traffic between your frontend webserver (Apache or Nginx) and the Netbox gunicorn process.

Hit Netbox with the request. There will be a bunch of garbage mixed in which is the TCP/IP headers, but buried in this you're expecting to see:

```
POST HTTP/1.0 /api/ipam/ip-addresses/
Host: netbox.example.com
... more headers
```

and then the JSON POST body content, possibly in a subsequent packet or packets.

**Reply by bbird81** [reply]
Hi, sorry for late reply, your comment helped a lot, it was an error in my script... sorry for this!
